packages/plugin-utils/src/options.ts

```diff
@@ -170,6 +175,7 @@ export interface ResolvedOptions {
   transformGroups: boolean
   sortUtilities: boolean
   safelist: Set<string>
+  onGenerated: Function
```

Comment:
Is this redundant?

Reply:
im very new to typescript :)
without it, im getting: 
Property 'onGenerated' does not exist on type 'UserOptions | ResolvedOptions'

Reply:
I miss read that. Somehow the collapsing on GitHub is quite confusing:
![image](https://user-images.githubusercontent.com/11247099/108868402-24ee3c80-7631-11eb-8d80-7d3d67121116.png)

No problem on this then.